.github/workflows/build-test-deploy.yaml

```diff
@@ -56,7 +56,32 @@ jobs:
           path: bin/
       - uses: engineerd/setup-kind@v0.5.0
       - run: chmod +x bin/preflight
-      - run: ./bin/preflight --interactive=false --format=json https://preflight.replicated.com
+      - run: |
+          ./bin/preflight --interactive=false --format=json https://preflight.replicated.com > result.json
+          cat result.json
```

Comment:
why is this needed?

Reply:
so we can see what's in that file.  because this test is kinds janky.